homeassistant/components/supla/__init__.py

```diff
@@ -55,9 +62,10 @@ def setup(hass, base_config):
 
```

Comment:
the library should allow passing in aiohttp websession so that it can use HAs shared session rather than creating its own.

Reply:
https://github.com/mwegrzynek/asyncpysupla/pull/1/

Reply:
OK, your suggestion makes sense, I merged your changes to asyncsupla. Can you point me to some examples of getting and passing shared websession to the library?

Reply:
```python
from homeassistant.helpers.aiohttp_client import async_get_clientsession

async_get_clientsession(hass)
```

Reply:
OK, done.